site/content/reference/api.md

```diff
@@ -1,9 +1,11 @@
 ---
-title: "API Reference"
-description: "NGINX Gateway API Reference"
+title: "API reference"
 weight: 100
 toc: false
 ---
+
+NGINX Gateway API Reference
```

Comment:
This is an autogenerated doc. The template is here: `site/config/api/pkg.tpl`

Reply:
I was trying to figure out how this was generated when I was looking at the unit tests. 

Should I remove this change from the PR entirely, or incorporate it by editing that template directly?

Reply:
You'll need to add it to the template, and then `make generate-api-docs` should just add what you added here.